src/jit/gtlist.h

```diff
@@ -46,7 +46,7 @@ GTNODE(CNS_STR          , "sconst"       ,GenTreeStrCon      ,0,GTK_LEAF|GTK_CON
 //-----------------------------------------------------------------------------
 
 GTNODE(NOT              , "~"            ,GenTreeOp          ,0,GTK_UNOP)
-GTNODE(NOP              , "nop"          ,GenTree            ,0,GTK_UNOP)
```

Comment:
OK - I can't remember why I felt the need to make this change, but I certainly don't think it needs to be part of this PR. I'll look into it further if changing this causes issues for the containedness stuff.